core/platform/taskqueue/cloud_taskqueue_services.py

```diff
@@ -91,9 +95,8 @@ def create_http_task(
         task['name'] = task_name
 
     # Use the CLIENT to build and send the task.
-    # Note: retry=None means that the default retry arguments in queue.yaml are
-    # used.
-    response = CLIENT.create_task(parent, task, retry=None)
+    # Note: retry=retry.Retry() means that the default retry arguments are used.
```

Comment:
Wait a sec -- one question about this. Does this mean that we no longer obey the stuff in queue.yaml? If so, that could be confusing if devs update queue.yaml and there is no effect there. Should we perhaps be setting this info in queue.yaml instead?

Reply:
I'm not entirnely sure whether the retry params in queue.yaml really work, since it seems that we have a lot of errors comming from this place, I want to try setting the retry here to see if that helps to reduce the number of errors that we get.